getsenic/senic-os
# ssh into dev image with non authorized key

Should resolve https://github.com/getsenic/senic-os/issues/120

Test:

* Build both the `senic-os` and `senic-os-dev`
* Confirm UART logging is working 
* Confirm you can't ssh to `senic-os`
* Confirm you can ssh to `senic-os-dev`
